release-notes/README.md

```diff
@@ -10,13 +10,32 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 * Added support for sending the click conversion A4T payload to Adobe Analytics for A4T-enabled Target activities when the `locationClicked` API is called.
 
+### React Native Assurance 2.0.0
+
+This major release introduces support for following:
+
+- Support for React Native version 0.60.+
+- Auto linking for native dependencies and removal of the bundled SDK binaries (XCFramework) from the React Native module.
+- Dynamic versions for native dependencies to always load the latest SDK.
+- Removal of registerExtension Javascript API.
+
 ## Aug 18, 2021
 
 ### iOS AEPCore 3.2.4
 
 * Fixed data race in `Event` and `ExtensionContainer` classes.
 * Fixed a memory leak in `EventHub.registerResponseListener`.
 
+### Flutter Assurance 2.0.0
+
+- Upgrade plugin to Flutter 2 and Null-safety support.
```

Comment:
Let's add the full version here with decimals. Is it 2.0?

Reply:
Oh, looks like it's "null safety" (without a dash)

Reply:
Let's do: "Upgrade plugin to support Flutter 2 and null safety support."  (with full decimal version for Flutter)

Reply:
Flutter 2.0 and Flutter 2 are interchangeably used by the community and google

https://developers.googleblog.com/2021/03/announcing-flutter-2.html
https://mobidev.biz/blog/flutter-2-for-web-new-features

Since the latest Flutter is 2.5. I would like to explicitly mention the flutter version that we support (i.e) Flutter 2.0